Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "bdk"
-version = "0.24.0"
+version = "0.27.0"
```

May as well update this as part of this PR:
```suggestion
version = "1.0.0-alpha"
```